fe1-web/store/reducers/__tests__/SocialReducer.test.ts

```diff
@@ -120,62 +122,79 @@ const filledState4 = {
   },
 };
 
-test('reducer should return the initial state', () => {
-  expect(socialReduce(undefined, {} as AnyAction))
-    .toEqual(emptyState);
-});
-
-test('reducer should add the first chirp correctly', () => {
-  expect(socialReduce(emptyState, addChirp(mockLaoId, chirp1)))
-    .toEqual(filledState1);
-});
-
-test('reducer should add the newer chirp before the first chirp', () => {
-  expect(socialReduce(filledState1, addChirp(mockLaoId, chirp2)))
-    .toEqual(filledState2);
-});
-
-test('reducer should add the newer chirp after the second chirp', () => {
-  expect(socialReduce(filledState2, addChirp(mockLaoId, chirp3)))
-    .toEqual(filledState3);
-});
-
-test('reducer should add the newest chirp on top', () => {
-  expect(socialReduce(filledState3, addChirp(mockLaoId, chirp4)))
-    .toEqual(filledState4);
-});
-
-test('selector should return an empty list of chirpState when no lao is opened', () => {
-  expect(makeChirpsList().resultFunc(emptyState, undefined))
-    .toEqual([]);
-});
-
-test('selector should return an empty list', () => {
-  socialReduce(undefined, {} as AnyAction);
-  expect(makeChirpsList().resultFunc(emptyState, mockLaoId))
-    .toEqual([]);
-});
-
-test('selector should return the first chirp state', () => {
-  socialReduce(emptyState, addChirp(mockLaoId, chirp1));
-  expect(makeChirpsList().resultFunc(filledState1, mockLaoId))
-    .toEqual([chirp1]);
-});
-
-test('selector should return the newer chirp before the first chirp', () => {
-  socialReduce(filledState1, addChirp(mockLaoId, chirp2));
-  expect(makeChirpsList().resultFunc(filledState2, mockLaoId))
-    .toEqual([chirp2, chirp1]);
-});
-
-test('selector should add the newer chirp after the second chirp', () => {
-  socialReduce(filledState2, addChirp(mockLaoId, chirp3));
-  expect(makeChirpsList().resultFunc(filledState3, mockLaoId))
-    .toEqual([chirp2, chirp3, chirp1]);
-});
-
-test('selector should return the newest chirp on top', () => {
-  socialReduce(filledState3, addChirp(mockLaoId, chirp4));
-  expect(makeChirpsList().resultFunc(filledState4, mockLaoId))
```

Comment:
I've also noticed the tests are not readable and modified the structure, in particular, separating the tests for reducer and the tests for selector... Maybe you can also separate these two. But I don't know which PR will be marged first😂.

Reply:
Ok, I've separated them.